src/Microsoft.AspNetCore.Sockets.Client/WebSocketsTransport.cs

```diff
@@ -38,12 +38,8 @@ public WebSocketsTransport(ILoggerFactory loggerFactory)
                 throw new ArgumentNullException(nameof(url));
             }
 
-            if (application == null)
-            {
-                throw new ArgumentNullException(nameof(application));
-            }
+            _application = application ?? throw new ArgumentNullException(nameof(application));
```

Comment:
Do we want to move to this pattern for param checking?
It's not consistent with how we do it right above